cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/PlatformParameters.java

```diff
@@ -123,6 +123,7 @@ default SpecialParameters specialParameters() {
         specialParameters.put(PlatformParametersConsts.VOLUME_ATTACHMENT_SUPPORTED, Boolean.TRUE);
         specialParameters.put(PlatformParametersConsts.VERTICAL_SCALING_SUPPORTED, Boolean.TRUE);
         specialParameters.put(PlatformParametersConsts.DB_SUBNETS_UPDATE_ENABLED, Boolean.FALSE);
+        specialParameters.put(PlatformParametersConsts.DELETE_VOLUMES_SUPPORTED, Boolean.TRUE);
```

Comment:
This parameter needs to be disabled in YarnPlatformParameters too.

Reply:
done